src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java

```diff
@@ -1129,15 +1129,18 @@ private boolean updateLookupInfoForProperties(List<Map<String, Object>> cbPlanMa
         boolean isUpdatedLookup = true;
         for (Map<String, Object> cbLookupInfo : cbPlanMap) {
             Date endDate = (Date) cbLookupInfo.get(Constants.END_DATE);
-            if (!planDto.getEndDate().equals(endDate)) {
+            Boolean currentIsApar = (Boolean) cbLookupInfo.get(Constants.IS_APAR);
+            Boolean newIsApar = planDto.getIsApar() != null ? planDto.getIsApar() : false;
+
+            if (!planDto.getEndDate().equals(endDate) || currentIsApar == null || !newIsApar.equals(currentIsApar)) {
```

Comment:
this approach is wrong... 

what should be done... 
boolean variable --> isAnyValueUpdated -- set this flag to true if any of the incoming value is different than existing value... also add the same into lookupInfoUpdated map
try to optimize the code to make sure individual attribute is compared with previous value and it is added into lookupInfoUpdated map -- instead of one single if condition